### ISSUE TITLE
Compatibilty issue with @quasar/apollo

### ISSUE THREAD

**Issue by lukadriel7** [quasarframework/app-extension-dotenv#33]
**Describe the bug**
Hello, I didn't know where to report so I guess I will do it on both repository.
@quasar/apollo extension has trouble using @quasar/dotenv variables when in quasar.extensions.json,
dotenv is configured before apollo. It adds some quotation marks around the variable causing issues on the GRAPHQL_URI.

changing the order fixes the problem but still very ennoying issue and difficult to debug.

**Steps to reproduce the behavior:**
1. Start new quasar project
2. Install @quasar/dotenv extension 
3. Install @quasar/apollo extension 
4. Set GRAPHQL_URI environment variable to a graphql URI
5. Make a graphql request on a page in the application
6. build the app and see the error in the network and console tab of the developper tools

**Expected behavior:**
No error no matter the order of installation

**Desktop:**
 - OS:Ubuntu 18.04
 - Browser chromium Version 85.0.4183.121 (Official Build) snap (64-bit)
 - Quasar v1.14.2, @quasar/quasar-app-extension-apollo v1.0.0-beta.6, @quasar/quasar-app-extension-dotenv v1.0.5

**Comment by hawkeye64** [comment]
@lukadriel7 
> and see the error in the network and console tab of the developper tools

What is the error? Is @quasar/apollo (I don't maintain that one) using @quasar/dotenv (I do maintain this one)?

**Comment by lukadriel7** [comment]
@hawkeye64 @quasar/apollo can be configured to uses environment variables in general (not just @quasar/dotenv).
By default it uses the GRAPHQL_URI variable or a an url if the variable is not set. Since I installed @quasar/dotenv before @quasar/apollo, I set the GRAPHQL_URI in my env files but I kept getting errors in my graphql call being performed on invalid url. Double quotes are added  to the variable GRAPHQL_URI content when @quasar/apollo is after @quasar/dotenv.

tl;dr: the error is the query uri becomes like http://localhost/%22http://example.com/graphql%22 causing it to fail

**Comment by hawkeye64** [comment]
@lukadriel7 Sorry, I don't really have a response to this, except to say uninstall the dotenv app-extension and use quasar.conf.js: `build -> env` to do the same thing. If it still happens, then it's not an issue with the app-extension.

**Comment by lukadriel7** [comment]
@hawkeye64 thanks, I just thought that people should be aware of this, I spent hours before realizing where the problem was
